### PR TITLE
Update CTA Button at Home Page Bottom

### PR DESCRIPTION
**Pull request by isralduke**
This PR:

- fixes the button alignment at the bottom of the home page

### CODE REVIEW

File: docs/index.njk [hunk 1]
```diff
@@ -205,7 +205,7 @@ title: Home
           <div class="col-10 col-md-6 col-lg-5 offset-lg-1">
               <p class="feature-big-text">Start using Pelican now.</p>
           </div>
-          <div class="col-10 col-md-6 col-lg-5 text-right">
+          <div class="col-10 col-md-6 col-lg-5 text-center text-lg-right">
```
Comment (frogpad7) [inline docs/index.njk:208]: Not sure if this makes much sense from what's in figma; currently at medium size the elements appear side-by-side, so why is the button centered then and then to the right at large?
Also, pretty sure the text above should get the same treatment, considering this is what it appears as:
image

**Review by frogpad7 — COMMENTED** [review]
I see some conflicting settings that I figured would be helpful to point out.

File: docs/index.njk (hunk 1 above)
Comment (isralduke) [reply docs/index.njk:208]: Good catch. thx

**Review by frogpad7 — APPROVED** [review]
Looks much cleaner now! Glad to see this and figma look very similar!

**Pull request merged**